### ISSUE TITLE
About  λ(bata_1)

### ISSUE THREAD

**Issue by woo2020** [PRIS-CV/Mutual-Channel-Loss#21]
Great work!But I have a little question about λ.
In your code:
`loss = ce_loss + args["alpha_1"] * MC_loss[0] +   args["beta_1"]  * MC_loss[1] `
And in your paper:
![image](https://user-images.githubusercontent.com/19606959/132094715-5b14adc9-f36b-4a30-a266-d10041a3b794.png)

So,I should set beta_1 as **-10** in code?
Please point out my problem if I misunderstood the operation here.Thanks a lot.

**Comment by dongliangchang** [comment]
Please ref #10 for details.

**Comment by woo2020** [comment]
I get it,thank you!
